test/unit/sql/add-column.test.js

```diff
@@ -69,6 +69,28 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
       });
     }
 
+    if (['oracle'].includes(current.dialect.name)) {
```

Comment:
Preferably we have the expectations of other dialects here as well, or is this functionality not available on other dialects?

Reply:
I have added expectations for other dialects.